wallet-test/src/test/scala/org/bitcoins/wallet/models/AddressDAOTest.scala

```diff
@@ -0,0 +1,29 @@
+package org.bitcoins.wallet.models
+
+import org.bitcoins.core.config.RegTest
+import org.bitcoins.core.crypto.ECPublicKey
+import org.bitcoins.core.crypto.bip44.{BIP44ChainType, BIP44Coin, BIP44Path}
+import org.bitcoins.core.protocol.P2SHAddress
+import org.bitcoins.core.util.CryptoUtil
+import org.bitcoins.wallet.util.BitcoinSWalletTest
+
+class AddressDAOTest extends BitcoinSWalletTest {
+  private val addressDAO = AddressDAO(dbConfig, chainParams)
+
+  it should "insert and read an address into the database" in {
```

Comment:
This test should crash, but doesn't... It should give an error message that the foreign key reference isn't found in the accounts table